### PR TITLE
Add a dockerfile to be able to create an image

### PR DESCRIPTION
**Pull request by danjenkins**
Let me know your thoughts on this

The image doesn't deal with config files as I would expect, if this becomes an automated build, people will base their own docker images on this one and add the config file into the working directory.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM golang
+
+WORKDIR /go/src/
+
+RUN go get github.com/nvisibleinc/go-ari-library
```
Comment (leifmadsen) [inline Dockerfile:5]: I would flatten these runs just so it doesn't add multiple unnecessary layers:

```
RUN mkdir go-ari-proxy \
  go get github.com/nvisibleinc/go-ari-library \
  go get golang.org/x/net/websocket
```

**Comment by leifmadsen** [comment]
Can you also update the README.md with some basic documentation on how to use and build with this? I think I even created a ToDo section for it... :)

**Comment by leifmadsen** [comment]
Additionally, is websocket a new requirement, or just one for your own usage here? Maybe it would be best to leave that out, and then I can figure out how to create an automated build perhaps, that you could use as a basis for your deploy?

**Comment by Marquis42** [comment]
The libs that a 'go get' is being run for are both explicit requirements in the go-ari-proxy.  However, from the standpoint of being 'lightweight' I'd rather have the binary prebuilt and just copied into a blank container (i.e., 'FROM scratch').  I'm probably going to open another PR with an approach along those lines, and we can discuss the merits of each and come to a consensus.

**Comment by leifmadsen** [comment]
Woah, apparently reactions are taking over the internet... I can use them on GitHub comments now! :)

(In some future, someone will read this like I'm an idiot. Of course reactions are everywhere. There was a time they weren't everywhere?)

@danjenkins Yea that sounds good to me :)  We can do it in another pull request too. I like what @Marquis42 is suggesting here. I planned to add auto-builds of the proxy as well to Docker Hub, but for some reason it wasn't playing nice with the code in an organization vs a user. I'll have to circle back around and see if I can fix that up.

**Comment by leifmadsen** [comment]
This is fine as is even though there are some changes we can refine. This gets it into the branch for someone to play with.

**Pull request merged**